Helios.xcodeproj/project.pbxproj

```diff
@@ -318,12 +322,19 @@
 		E0D079DE0996912E00B2C3F4 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ARCHS = "$(ARCHS_STANDARD)";
+				CLANG_CXX_LANGUAGE_STANDARD = "c++17";
+				CLANG_CXX_LIBRARY = "libc++";
 				COPY_PHASE_STRIP = NO;
+				CURRENT_PROJECT_VERSION = 1.51;
 				GCC_DYNAMIC_NO_PIC = NO;
 				GCC_ENABLE_FIX_AND_CONTINUE = YES;
 				GCC_GENERATE_DEBUGGING_SYMBOLS = YES;
 				GCC_OPTIMIZATION_LEVEL = 0;
+				"GCC_VERSION[sdk=macosx10.4][arch=*]" = "";
 				INFOPLIST_FILE = "Info-Helios.plist";
+				MACOSX_DEPLOYMENT_TARGET = 10.14;
```

Comment:
Why 10.14? libc++ was introduced in 10.9, so that should be the new deployment target.